src/main/java/com/crud/tasks/service/SimpleEmailService.java

```diff
@@ -28,9 +30,10 @@ public void send(final Mail mail) {
 
     private SimpleMailMessage createMailMessage(final Mail mail) {
         SimpleMailMessage mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(mail.getMailTo());
+        mailMessage.setTo(Optional.ofNullable(mail.getMailTo()).orElse("Email address incorrect"));
```

Comment:
czemu tak? może by tak rzucić wyjątek jak mail jest nullem? obecnie jak mail jest nullem to robisz `setTo("Email address incorrect")`